packages/audio/pulseaudio/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="pulseaudio"
-PKG_VERSION="13.0"
-PKG_SHA256="961b23ca1acfd28f2bc87414c27bb40e12436efcf2158d29721b1e89f3f28057"
+PKG_VERSION="13.99.2"
```

Comment:
This looks like a pre-release version. Please drop.

Reply:
its basically 14.0 RC2, guess we can wait for the final one

Reply:
I’ll remove this from this pull request. Below is the draft release notes as FYI. What is the best way to track 14.0? A draft pull request or an issue or neither? Is it something to have in the target release for le10. 

https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/1007

Reply:
@lrusak - pulse audio dropped :-)

Reply:
Pull Rebased